Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -238,6 +233,23 @@ public struct Configuration {
     }
 }
 
+func errorMessage(for error: Error, hasCustomConfigurationFiles: Bool) -> String? {
+    switch error {
+    case let ConfigurationError.initialFileNotFound(path):
+        if hasCustomConfigurationFiles {
+            return "SwiftLint Configuration Error: Could not read file at path: \(path)"
+        } else {
+            return nil
```

Comment:
As the semantics of this function are probably a bit unclear at first glance, it would be great to have a comment above the `return nil` line explaining why `nil` is returned, e. g. `// The initial configuration file wasn't found, but the user didn't explicitly specify one -> don't handle as error`